databridge.component/src/test/java/org/eclipse/digitaltwin/basyx/databridge/executable/regression/TestDataBridgeAASPollingConsumer.java

```diff
@@ -27,8 +27,7 @@
 
 import java.io.IOException;
 import java.util.UUID;
-import org.eclipse.basyx.aas.aggregator.api.IAASAggregator;
-import org.eclipse.basyx.aas.aggregator.proxy.AASAggregatorProxy;
+
```

Comment:
Remove line

Reply:
This new line indicates the border between java standard library imports and maven dependencies
-> Boy scout Rule